lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -445,6 +451,14 @@ private class MutationExtension<T : Any>(val spec: MutationExtensionSpec<T>) : E
       lastActionState = actionState
     }
 
+    private fun dataItemDragFlags(): (T) -> Int {
```

Comment:
This is the magic sauce. Everything else in this PR is about updates to sample app